web/frontend/src/components/application/ServicesBox.tsx

```diff
@@ -1,21 +1,66 @@
 import React, { useState, useEffect } from 'react'
 import { OPACITY_100, OPACITY_15, RICH_BLACK, WHITE, TRANSPARENT, MEDIUM, SMALL, BLACK_RUSSIAN } from '@platformatic/ui-components/src/components/constants'
+import { ApiReferenceReact } from '@scalar/api-reference-react'
 import styles from './ServicesBox.module.css'
 import typographyStyles from '../../styles/Typography.module.css'
 import commonStyles from '../../styles/CommonStyles.module.css'
 import { BorderedBox } from '@platformatic/ui-components'
 import Icons from '@platformatic/ui-components/src/components/icons'
 import useAdminStore from '../../useAdminStore'
-import { getServices } from '../../api'
+import { getOpenApi, getServices } from '../../api'
 import { ServiceData } from 'src/types'
+import { HOME_PATH } from '../../ui-constants'
+import { useNavigate } from 'react-router-dom'
+
+interface ServiceDetailPanelProps {
+  openapi: unknown;
+  onClose: () => void;
+}
+
+function ServiceDetailPanel ({ openapi, onClose }: ServiceDetailPanelProps): React.ReactElement {
+  const customCss = `
+  .dark-mode {
+    --scalar-color-1: rgb(255 255 255 / var(--tw-bg-opacity));
+    --scalar-background-1: rgb(0 5 11 / var(--tw-bg-opacity));
+    --scalar-background-3: rgb(0 5 11 / var(--tw-bg-opacity));
+    --sidebar-background-1: rgb(0 5 11 / var(--tw-bg-opacity));
+    --sidebar-border-color: rgb(0 5 11 / var(--tw-bg-opacity));
+  }
+  .darklight {
+    display: none !important;
+  }
+  .contents > section,
+  .section-container > section {
+    padding: 40px 0 !important;
+  }`
+
+  return (
+    <div className={styles.serviceDetailPanel}>
+      <div className={`${commonStyles.smallFlexRow} ${commonStyles.fullWidth} ${commonStyles.justifyBetween}`}>
+        <div />
+        <button
+          onClick={onClose}
+          className={styles.closeButton}
+        >
+          <Icons.CloseIcon color={WHITE} size={MEDIUM} />
+        </button>
+      </div>
+
+      {openapi && typeof openapi === 'string' && openapi !== 'null'
+        ? <ApiReferenceReact configuration={{ content: openapi, showSidebar: false, customCss }} />
+        : <div className={`${typographyStyles.desktopBodyLargeSemibold} ${typographyStyles.textWhite} ${typographyStyles.ellipsis} ${typographyStyles.textCenter} ${styles.emptyOpenApi}`}>No valid Open API data available</div>}
+    </div>
```

Comment:
I think we need a slightly better messaging in case there is no OpenAPI for the service, or possibly just a relatively simple UI to send fetch() calls.